bluebottle/activities/triggers.py

```diff
@@ -266,22 +287,45 @@ class TeamTriggers(TriggerManager):
                     'members',
                     ParticipantStateMachine.accept,
                     conditions=[needs_review]
-                )
+                ),
+                RelatedTransitionEffect(
+                    'activity',
+                    TimeBasedStateMachine.lock,
+                    conditions=[team_activity_will_be_full]
+                ),
             ]
         ),
 
         TransitionTrigger(
             TeamStateMachine.cancel,
             effects=[
-                TeamContributionTransitionEffect(ContributionStateMachine.fail),
-                NotificationEffect(TeamCancelledMessage),
-                NotificationEffect(TeamCancelledTeamCaptainMessage)
+                # RelatedTransitionEffect(
+                #     'members',
+                #     ParticipantStateMachine.remove
+                # ),
+                RelatedTransitionEffect(
+                    'activity',
+                    TimeBasedStateMachine.reopen,
+                    conditions=[team_activity_will_not_be_full]
+                ),
+                # TeamContributionTransitionEffect(ContributionStateMachine.fail),
+                # NotificationEffect(TeamCancelledMessage),
+                # NotificationEffect(TeamCancelledTeamCaptainMessage)
```

Comment:
Just remove stuff it is no longer needed

Reply:
It _was_ needed :-) Comment it out for testing purposes. Now it's back